packages/web/test/src/lib/walletconnect-mode.ts

```diff
@@ -0,0 +1,111 @@
+import WalletConnect from "@walletconnect/client";
+import algosdk, { Account, Transaction } from "algosdk";
+import assert from "assert";
+import QRCodeModal from "algorand-walletconnect-qrcode-modal";
+import { testnetURL, types, WallectConnectSession } from "../../../src";
+import { algoexplorerAlgod, getSuggestedParams } from "../../../src/lib/api";
+import { HttpNetworkConfig } from "../../../src/types";
+
+describe("Webmode - Wallet Connect test cases ", function () {
+    let connector: WallectConnectSession;
+    let sender: Account;
+    let receiver: Account;
+
+    let algodClient: algosdk.Algodv2;
+    const walletURL: HttpNetworkConfig = {
+        token: "",
+        server: testnetURL,
+        port: "",
+    }
+    algodClient = algoexplorerAlgod(walletURL);
+
+    this.beforeEach(function () {
+        sender = algosdk.generateAccount();
+        receiver = algosdk.generateAccount();
+        connector = new WallectConnectSession(walletURL, new WalletConnect({
+            bridge: "https://bridge.walletconnect.org",
+            qrcodeModal: QRCodeModal,
+        }));
+    });
+
+    it("Should executeTx without throwing an error", function () {
```

Comment:
```suggestion
    it("Should executeTx not throw an error", function () {
```

Reply:
I think without throwing makes more sense

Reply:
Think about it once before merge.

Reply:
updated it with `Should run executeTx function without throwing an error`